app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/ResourceResolver.java

```diff
@@ -1,10 +1,48 @@
 package io.github.droidkaigi.confsched2017.view.helper;
 
+import android.content.Context;
 import android.support.annotation.StringRes;
 
-public interface ResourceResolver {
+import java.io.IOException;
+import java.io.InputStream;
 
-    String getString(@StringRes int resId);
+import javax.inject.Inject;
+import javax.inject.Singleton;
 
-    String getString(@StringRes int resId, Object... formatArgs);
+import timber.log.Timber;
+
+@Singleton
+public class ResourceResolver {
+
+    private static final String TAG = ResourceResolver.class.getSimpleName();
+
+    private final Context context;
+
+    @Inject
+    public ResourceResolver(Context context) {
+        this.context = context;
+    }
+
+    public String getString(@StringRes int resId) {
+        return context.getString(resId);
+    }
+
+    public String getString(@StringRes int resId, Object... formatArgs) {
+        return context.getString(resId, formatArgs);
+    }
+
+    public String loadJSONFromAsset(final String jsonFileName) {
```

Comment:
We have another option that split this class into `ResourceResolver` and `AssetsResolver`.
However I think it's little redundant.
What do you think about?